dev/CommandBarFlyout/InteractionTests/TextCommandBarFlyoutTests.cs

```diff
@@ -480,11 +475,19 @@ public void ValidateKeyboarding()
                 {
                     Log.Comment("Use Escape to close the context menu. The TextBox should now have focus.");
                     KeyboardHelper.PressKey(Key.Escape);
+
+                    // On 19H1, there's a bug with the focus restoration code, so we'll manually restore focus to allow the rest of the test to run.
+                    if (PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.NineteenH1))
+                    {
+                        textBox.SetFocus();
+                    }
+
                     waiter.Wait();
                 }
                 
                 Log.Comment("Give focus to the RichEditBox.");
-                FocusHelper.SetFocus(FindElement.ById("RichEditBox"));
```

Comment:
>FocusHelper.SetFocus(FindElement.ById("RichEditBox")); [](start = 16, length = 54)

What was the motivation for splitting this line?

Reply:
I needed to reference richEditBox later on line 525, so I needed to store it in a variable.